### PR TITLE
docs: fix small typos

### PR DESCRIPTION
**Pull request by smitssjors**

### CODE REVIEW

File: docs/toc.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const toc: RawTableOfContents = {
           ["forms", "Forms", "link:latest"],
           ["islands", "Interactive islands", "link:latest"],
           ["static-files", "Static files", "link:latest"],
-          ["middleware", "Middlewares", "link:latest"],
+          ["middleware", "Middleware", "link:latest"],
```
Comment (marvinhagemeister) [inline docs/toc.ts]: Let's keep the visible page title as `Middlewares`. All other things in the docs are in plurals too. Rather we should redirect the url to the plural version at some point, but that can happen in another PR.

File: docs/latest/concepts/middleware.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ be used to pass arbitrary data to downstream (or upstream) handlers. This
 special [\_app](/docs/concepts/app-wrapper.md) wrapper and normal
 [routes](/docs/concepts/routes.md). `ctx.state` is normally set by modifying its
 properties, e.g. `ctx.state.loggedIn = true`, but you can also replace the
-entire object like `ctx.state = { loggedIn = true }`.
+entire object like `ctx.state = { loggedIn: true }`.
```
Comment (marvinhagemeister) [inline docs/latest/concepts/middleware.md:18]: Great catch!

File: docs/toc.ts (hunk 1 above)
Comment (smitssjors) [reply docs/toc.ts]: I did not consider that they were all plural, I've reverted the change.

**Review by marvinhagemeister — APPROVED** [review]
Awesome, thanks for fixing this 👍

**Pull request merged**